tests/integration/utils.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from collections import namedtuple
+
+import requests
+from requests_kerberos import HTTPKerberosAuth
+
+
+Response = namedtuple("Response", "data id status")
+
+
+class Client:
+    """Simplifies access to the Cachito API."""
+
+    def __init__(self, cachito_api_url):
+        """
+        Initialize the Client class.
+
+        :attribute str _cachito_api_url: URL of the Cachito API
+        """
+        self._cachito_api_url = cachito_api_url
+
+    def fetch_request(self, request_id):
+        """
+        Fetch a request from the Cachito API.
+
+        :param int request_id: ID of the request in Cachito
+        :return: Object that contains response from the Cachito API
+        :rtype: Response
+        """
+        resp = requests.get(f"{self._cachito_api_url}/requests/{request_id}")
+        resp.raise_for_status()
+        return Response(resp.json(), resp.json()["id"], resp.status_code)
+
+    def create_new_request(self, auth, payload):
```

Comment:
Why did you decide to not have authentication automatically configured using the configuration rather than having to pass it in all the time?

It's okay that you did but I'm just wondering if there is a reason that this needs to be explicit.

Reply:
There's no reason, I didn't realise I could do it like that. It's changed now.